static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -98,20 +122,28 @@ const PanelBody = styled(BasePanelBody)`
   overflow-y: auto;
 `;
 
-const PanelItemCenter = styled(PanelItem)`
+const PanelItemCenter = styled(PanelItem)<PanelItemCenterProps>`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  border-bottom: none;
+  border-left: 4px solid transparent;
   padding: ${space(1)} ${space(1.5)};
   cursor: pointer;
+  &:last-child {
+    border-left: 4px solid transparent;
+  }
```

Comment:
this looks redundant
```suggestion
```

Reply:
although it looks redundant in code, the component `PanelItem` has a last-child, which overrides the style and the last element seems misaligned.

Reply:
You know what, can we put this border on another inner element?

When the border is on the `PanelItem` the bottom edge gets crooked because it's melding in with the gray border-bottom.

<img width="349" alt="Screen Shot 2022-05-12 at 12 25 56 AM" src="https://user-images.githubusercontent.com/187460/167900350-f6e62b32-9672-4a5a-bd0f-ee7f38d53467.png">